examples/charts/steps.py

```diff
@@ -1,9 +1,6 @@
 from collections import OrderedDict
 
-import numpy as np
-import pandas as pd
-from bokeh.charts import Chart, Step, Line
-from bokeh.plotting import show,output_file
+from bokeh.charts import Step, show, output_file
```

Comment:
Ideally users would import `show`, etc. from state, but we now have its of code that imports it from `plotting`. As a compromise I'd rather have both `plotting` and `charts` import `show` etc, and then users can use `show` from either. The main point is to avoid "mixing" the APIs unnecessarily. If you just want a simple chart its confusing to users to have to import things from the `plotting.py` API as well. Eventually maybe we can deprecate the usage from `plotting.py` and `charts.py` and move everyone to importing from `state.py`